paths.go

```diff
@@ -44,12 +44,11 @@ func newPaths(os, arch string, o Options) (p *Paths, err error) {
 	p.initAstilectronDirectory()
 	p.astilectronApplication = filepath.Join(p.astilectronDirectory, "main.js")
 	p.astilectronDownloadSrc = AstilectronDownloadSrc()
-	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-v%s.zip", VersionAstilectron))
+	p.astilectronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("astilectron-%s-%s-v%s.zip", os, arch, VersionAstilectron))
 	p.astilectronUnzipSrc = filepath.Join(p.astilectronDownloadDst, fmt.Sprintf("astilectron-%s", VersionAstilectron))
 	p.electronDirectory = filepath.Join(p.vendorDirectory, "electron")
 	p.electronDownloadSrc = ElectronDownloadSrc(os, arch)
-	// TODO Split folders for each OS/ARCH couples?
-	p.electronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("electron-v%s.zip", VersionElectron))
+	p.electronDownloadDst = filepath.Join(p.vendorDirectory, fmt.Sprintf("electron-%s-%s-v%s.zip", os, arch, VersionElectron))
```

Comment:
You need to update tests in `paths_test.go` line 28 and 30 as well since this change breaks them

Also you'll need to add tests to make sure `p.electronDownloadDst` is set correctly for any os/arch (also in `paths_test.go`)